README.md

```diff
@@ -49,4 +49,8 @@ yarn preview
 - [x] Mobile optimization
 - [x] Edge runtime
 
-## Testing 
\ No newline at end of file
+## Cypress Testing 
+To run the tests use the following command
+`npx cypress run`
```

Comment:
Can we add that the user needs to have the server running first with yarn preview?